ako-infra/ingestion/avi_infra_controller.go

```diff
@@ -84,23 +84,92 @@ func (a *AviControllerInfra) VerifyAviControllerLicense() error {
 	return nil
 }
 
+func (a *AviControllerInfra) checkNSAnnotations() (error, string) {
+	cloudName := ""
+	nsName := utils.GetAKONamespace()
+	nsObj, err := a.cs.CoreV1().Namespaces().Get(context.TODO(), nsName, metav1.GetOptions{})
+	if err != nil {
+		utils.AviLog.Errorf("Failed to GET the %s namespace details due to the following error :%v", nsName, err.Error())
+		return err, cloudName
+	}
+	if nsObj.Annotations != nil {
```

Comment:
nit: no need for a nil check. Also, let's create a variable for `ako.vmware.com/wcp-cloud-name` string